.github/workflows/update-gardenlogin.yaml

```diff
@@ -0,0 +1,34 @@
+name: gardenlogin-updater
+
+on:
+  release:
+    types:
+      - published
+jobs:
+  update_gardenlogin_in_homebrew_tap:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build the binary-files
+        id: build_binary_files
+        run: |
+          sudo apt-get update
+          sudo apt-get install make -y
+          make build
+          echo ::set-output name=latest_release_filtered_tag::${GITHUB_REF##*/}
+      - name: Upload binaries to release
+        uses: AButler/upload-release-assets@v2.0
+        with:
+          files: 'bin/darwin-amd64/gardenlogin_darwin_amd64;bin/linux-amd64/gardenlogin_linux_amd64'
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          release-tag: ${{ steps.build_binary_files.outputs.latest_release_filtered_tag }}
+      - name: Send update with latest versions to gardener/homebrew-tap
+        run: |
+          mac_sha256sum=$(sha256sum bin/darwin-amd64/gardenlogin_darwin_amd64 | awk '{print $1}')
+          linux_sha256sum=$(sha256sum bin/linux-amd64/gardenlogin_linux_amd64 | awk '{print $1}')
+          data='{"event_type": "update", "client_payload": { "component": "gardenlogin", "tag": "'"${{ steps.build_binary_files.outputs.latest_release_filtered_tag }}"'", "mac_sha": "'"$mac_sha256sum"'", "linux_sha": "'"$linux_sha256sum"'"}}'
+          echo "${data}"
```

Comment:
Is this echo statement supposed to be here? Or is it for debugging only?

Reply:
it's for debugging in case something went wrong with the action